src/server/permissions/checkers/AppCloudBridge.ts

```diff
@@ -5,7 +5,7 @@ import { AppPermissions } from '../AppPermissions';
 export const AppCloudBridge = {
     getWorkspaceToken(scope: string, appId: string): void {
         const grantedPermission = AppPermissionManager.hasPermission(appId, AppPermissions.cloud['workspace-token']);
-        const isMissingScope = grantedPermission?.scopes?.includes(scope) === false;
+        const isMissingScope = grantedPermission && grantedPermission.scopes && grantedPermission.scopes.includes(scope) === false;
```

Comment:
![image](https://user-images.githubusercontent.com/32427260/114172829-c96dd900-9968-11eb-9ded-282c2b03738e.png)

I cannot push because tslint cannot recognize the latest optional chainning syntax. Can you please check it with `npm run lint` in your local?